cmd/config-reloader/main.go

```diff
@@ -45,8 +45,11 @@ func main() {
 		// management APIs, e.g.
 		// https://prometheus.io/docs/prometheus/latest/management_api/
 		// https://prometheus.io/docs/alerting/latest/management_api/
-		reloadURLStr  = flag.String("reload-url", "http://127.0.0.1:19090/-/reload", "reload endpoint triggers a reload of the configuration file")
-		readyURLStr   = flag.String("ready-url", "http://127.0.0.1:19090/-/ready", "ready endpoint returns a 200 when ready to serve traffic")
+		reloadURLStr                      = flag.String("reload-url", "http://127.0.0.1:19090/-/reload", "reload endpoint of the configuration target that triggers a reload of the configuration file")
+		readyURLStr                       = flag.String("ready-url", "http://127.0.0.1:19090/-/ready", "ready endpoint of the configuration target that returns a 200 when ready to serve traffic")
+		readyProbingInterval              = flag.Duration("ready-probing-interval", 1*time.Second, "how often to poll ready URL during start-up")
```

Comment:
suggestion (nit, non-blocking): rename flags to "startup-probing-interval" and "startup-probing-no-connection-threshold"?

That reads more clearly to me that these parameters only control startup behavior, rather than continually probing during the lifetime of the process.

Reply:
addressed (moved to `ready-startup-probing-interval` and `ready-startup-probing-no-conn-threshold`)